### DISCUSSION TITLE
Safe to install on a MacOS "Server" machine?

### DISCUSSION THREAD

**Discussion by tcmeyers** [Homebrew/discussions#3887] [Getting started]
I have a web server running on macOS Sierra 10.12.6 with the old Apple "Server" app 5.3.1. I have purposely not moved on because later Server dropped features I need (the Web Server part).

My pages are heavily PHP based, again, old, 5.6.30.

I'm wanting to install the PHP imagick extension to deal with images better than the GD I'm using. I've been recommend to use Homebrew to do this.  First I have to install Homebrew.

It's nice that Homebrew tells you what it's going to do! But I don't know enough to understand the consequences.

I put this in Terminal:
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

And, this is what it said, among other things:

==> You are using macOS 10.12.
==> We (and Apple) do not provide support for this old version.
This installation may not succeed.
After installation, you will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving any
issues you experience while you are running this old version.

So, I'm not supposed to ask here, but I don't understand what it's telling me. 

Can anyone briefly explain? It's telling me I shouldn't do it?

Troy

**Comment by SMillerDev** [comment]
It's telling you Homebrew won't work on macOS Sierra.

**Reply by tcmeyers** [reply]
Thanks, smillerdev. Is it also saying that there aren't prior versions that would?

**Reply by SMillerDev** [reply]
No, Homebrew is a rolling release package manager so older versions don't work anymore.

**Reply by tcmeyers** [reply]
Thank you! I'm looking at MacPorts now, seems like there is still a Sierra version.

**Comment by Cunfuu** [comment]
if you ever managed to use it on  10.12.6 the thing is you have to rely on GitHub repo's and you have to know the exact repo you are about to install its kind of a complicated answer but yes it has some vulnerabilities
